.github/actions/report-run-failure/README.md

```diff
@@ -1,7 +1,15 @@
+# Report GitHub workflow run failure to Slack
+
+This GitHub Action sends a notification to a nominated Slack channel when a GitHub workflow run fails. The action is typically used to notify a team about critical CI failures on the main branch. It helps to ensure that key issues are addressed quickly by sending relevant information including a direct link to the build logs.
+
 ## Notes on using this action
 
-  - This action relies on GOVUK_SLACK_WEBHOOK_URL actions secret being configured for the repository. It’s added to all GOV.UK repositories as an organisation secret in the [GOV.UK GitHub Infrastructure configuration](https://github.com/alphagov/govuk-infrastructure/blob/main/terraform/deployments/github/main.tf)
-  - To minimise noise, the slack message is only sent out for failures on the main branch.
+- It is configured with the following inputs:
+  -  `slack_webhook_url`: (required) The Slack webhook URL used to send notifications to a specific Slack channel. It stored as a GitHub secret (GOVUK_SLACK_WEBHOOK_URL), which is added to all GOV.UK repositories as an organisation secret in the [GOV.UK GitHub Infrastructure configuration](https://github.com/alphagov/govuk-infrastructure/blob/main/terraform/deployments/github/main.tf).
```

Comment:
Is this secret added to all GOV.UK repos, and will it be added if someone creates a new repo?

Reply:
1. yes, Added here: https://github.com/alphagov/govuk-infrastructure/pull/1875
2. yes, if the [new repo guide](https://docs.publishing.service.gov.uk/manual/github-new-repo.html) is followed and  terraform applied

Reply:
Good idea: https://github.com/alphagov/govuk-developer-docs/pull/5041